### PR TITLE
adding and integrating the html and css

### PR DESCRIPTION
**Pull request by Gedewon**
### In this milestone, I:
- Built the Hello Microverse Project
- Used linters for code standard

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse</title>
+</head>
+<body>
+    <h1>microverse project-1</h1>
+</body>
```
Comment (thecodechaser) [inline index.html:10]: - [ ] Good job here using the `<h1>` tag for heading but according to the project requirements the `<h1>` should have the heading "Hello Microverse!" instead "microverse project-1" , Please change your `<h1>` heading with "Hello Microverse!"

**Review by thecodechaser — CHANGES_REQUESTED** [review]
Hi @Gedewon 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a title and summary. :heavy_check_mark:
- test.md file is mentioned in .gitignore file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._
- [ ]  Please add a descriptive README file to your project, Feel free to use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ]  Please update your `<h1>` heading, check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Gedewon) [reply index.html:10]: okay on it

**Review by Whoistolu — APPROVED** [review]
### Hi @Gedewon,

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**